asv_bench/benchmarks/benchmarks.py

```diff
@@ -314,6 +317,33 @@ def time_apply(self, shape, axis):
     def time_mean(self, shape, axis):
         execute(self.df.mean(axis=axis))
 
+    def time_mode(self, shape, axis):
+        execute(self.df.mode(axis=axis))
+
+    def time_add(self, shape, axis):
+        execute(self.df.add(2, axis=axis))
+
+    def time_mul(self, shape, axis):
+        execute(self.df.mul(2, axis=axis))
+
+    def time_mod(self, shape, axis):
+        execute(self.df.mod(2, axis=axis))
+
+    def time_abs(self, shape):
+        execute(self.df.abs())
+
+    def time_aggregate(self, shape, axis):
+        execute(self.df.aggregate(lambda df: df.sum(), axis=axis))
+
+    def value_counts(self, shape):
```

Comment:
We already bench the function: https://github.com/modin-project/modin/blob/b7027597985a442734a57949d72e3fa0625b4d43/asv_bench/benchmarks/benchmarks.py#L477